projects/qgrid-core/column-type/date.column.js

```diff
@@ -21,6 +24,14 @@ export class DateColumnModel extends DataColumnModel {
 		super('date');
 
 		this.format = 'MM/dd/yyyy';
+
+		this.label = function (row) {
+			const value = getValue(row, this);
+			const parse = parseFactory('date');
```

Comment:
please, move it from this function, it's not required to be inside

Reply:
updated